### ISSUE TITLE
Database already populated with incompatible schema, options, or version

### ISSUE THREAD

**Issue by jakubgs** [sourcecred/sourcecred#1399]
I'm getting this error:
```
Error: Database already populated with incompatible schema, options, or version                                                                               
    at _inTransaction (webpack:///./src/graphql/mirror.js?:163:49)
    at _inTransaction (webpack:///./src/graphql/mirror.js?:572:142)
    at Mirror._initialize (webpack:///./src/graphql/mirror.js?:159:180)
    at new Mirror (webpack:///./src/graphql/mirror.js?:61:701)
    at fetchGithubRepo (webpack:///./src/plugins/github/fetchGithubRepo.js?:38:239)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
Despite running the same commit(1ff13b266ed7c9f8f1860274f3832fc91d66839c) that I used for the previous run.

The only thing that changed is that I modified the `BLACKLISTED_IDS` list in [`src/plugins/github/blacklistedObjectIds.js`](https://github.com/sourcecred/sourcecred/blob/master/src/plugins/github/blacklistedObjectIds.js) which as far as I know should not affect the database schema.

**Comment by decentralion** [comment]
The BLACKLISTED_IDS are included in the schema, to prevent getting into inconsistent load states. I proposed a pull (#1286) that would make the system more permissive, but it didn't merge.

You can fix it for yourself by removing the GItHub cache, e.g. via `rm -r $SOURCECRED_DIRECTORY/cache`

**Comment by jakubgs** [comment]
Yeah, I know I can remove GH cache, but that means I have to restart the whole pulling process from scratch, and then I hit my face against #1352.

Shouldn't the blacklist be a dynamic parameter rather than part of the DB schema. Kinda weird.

**Comment by decentralion** [comment]
If you think #1286 will make your life easier we can revive and merge it.
